src/mono/msbuild/apple/build/AppleBuild.InTree.targets

```diff
@@ -12,6 +12,15 @@
   <Import Project="..\..\common\LibraryBuilder.targets" />
   <Import Project="$(MSBuildThisFileDirectory)AppleBuild.targets" />
 
+  <!-- Workaround for SDK not knowing about CoreCLR iOS runtime packs -->
+  <Target Name="UpdateKnownFrameworkReference" BeforeTargets="ProcessFrameworkReferences">
```

Comment:
Should we condition this to run when targeting Apple mobile only?

Reply:
The whole file is already condition for that, isn't it?